src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,23 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.book.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector
+            .getInstance("mate.academy.lib");
+    private static final BookDao dao = (BookDao) injector.getInstance(BookDao.class);
```

Comment:
The `BookDao` interface specifies that the `findById` method should return an `Optional<Book>`, but the current code calls `dao.findById(1L)` without handling the `Optional`. Consider using `Optional` methods like `ifPresent`, `orElse`, or `orElseThrow` to handle the result properly.